wayfinder/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-wayfinder"
+description = "The wayfinder check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.8"
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "wayfinder",
+]
+authors = [
+    { name = "Jonny Scott", email = "jonny.scott@appvia.io" },
```

Comment:
You may want these to be company names and emails; but I'll leave that up to you.

Reply:
Updated to use company details